### PR TITLE
Some random minor improvements.

### PR DESCRIPTION
**Pull request by taphu**
See commits for comments.

### CODE REVIEW

File: src/Web/Moonshine.hs [hunk 1]
```diff
@@ -190,10 +218,12 @@ runMoonshine (Moonshine m) = do
   setupLogging systemConfig
   metricsStore <- EkgMetrics.newStore
   (State {snap}, val) <- m State {userConfig, snap = return (), metricsStore}
-  startServer (serverConfig systemConfig) metricsStore snap
+  let snap2 = snap <|> notFound
```
Comment (potomak) [inline src/Web/Moonshine.hs]: Does this mean that we can get rid of https://github.com/SumAll/haskell-sync-worker-framework/blob/master/src/Sync/Worker.hs#L81 introduced by https://github.com/SumAll/haskell-sync-worker-framework/pull/4 ?

Comment (glasserc) [inline src/Web/Moonshine.hs]: I think so. @taphu ?

Comment (taphu) [inline src/Web/Moonshine.hs]: Yes.

**Comment by taphu** [comment]
Ok, I 86'd the iffy commit, and added an additional commit for `setServerVersion`.

**Pull request merged**